test/bad_test_confs/nc96ads_a100_v4.conf

```diff
@@ -25,11 +25,18 @@
  * || check_hw_eth lo
  * || check_hw_eth eth0
  * || check_hw_eth docker0
+ * || check_hw_topology
 
 
 ########################################################
 ####
 #### GPU checks
 ####
- * || check_gpu_bw 23 250
-
+#  * || check_gpu_count 4
+#  * || check_gpu_xid
+#  * || check_nvsmi_healthmon
+ * || check_gpu_bw 50 250
+#  * || check_gpu_ecc 20000000 10000
+#  * || check_gpu_clock_throttling
```

Comment:
clean up these commented out lines?

Reply:
I have been going back and forth whether to keep them there or not. Maybe its best that the bad files only contain the failing test. Let me address.

Reply:
addressed